### ISSUE TITLE
ipmi-host: example fru_config.yaml is used for non-MRW machines

### ISSUE THREAD

**Issue by mine260309** [openbmc/openbmc#3134]
While building `phosphor-ipmi-host.bb`, `fru_config.yaml` is needed:
```
FRU_YAML_GEN=${STAGING_DIR_NATIVE}${config_datadir}/fru_config.yaml
```

This config comes from:
1. `phosphor-ipmi-fru-merge-config-native.bb` merges `virtual/phosphor-ipmi-fru-read-inventory` and `virtual/phosphor-ipmi-fru-read-bmc-inventory` to `fru_config.yaml`
2. `virtual/phosphor-ipmi-fru-read-inventory` is provided by:
   * `phosphor-ipmi-fru-read-inventory-example-native.bb`
   * `phosphor-ipmi-fru-read-inventory-mrw-native.bb`

* For MRW machines, it's using `phosphor-ipmi-fru-read-inventory-mrw-native.bb` and it generates the config yaml from MRW and `phosphor-ipmi-fru-hostfw-config`, which is OK.
* For non-MRW machines, it's using `phosphor-ipmi-fru-read-inventory-example-native.bb` and the config yaml is an example one in `phosphor-ipmi-host`.

So the build for all non-MRW machines is using the example fru config, it is wrong.

**Comment by mine260309** [comment]
Actually it can be simply done by re-use the same yaml config for different recipes:
https://gerrit.openbmc-project.xyz/#/c/10497/

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/10497> meta-romulus: Add ipmi-fru-read-inventory config
Resolves:  #3134 ipmi-host: example fru_config.yaml is used for non-MRW machines
